sdk/core/Azure.Core.Experimental/src/ClassifiedResponse.cs

```diff
@@ -0,0 +1,88 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Diagnostics.CodeAnalysis;
+using System.IO;
+using System.Text;
+
+namespace Azure.Core
+{
+    /// <summary>
+    /// Wrap Response and add IsError field.
+    /// </summary>
+    public class ClassifiedResponse : Response
+    {
+        private bool _disposed;
+
+        private Response Response { get; }
+
+        /// <summary>
+        /// </summary>
+        public bool IsError { get; private set; }
+
+        internal void EvaluateError(HttpMessage message)
```

Comment:
Do this in ClassifiedResponse  ctor.

Reply:
Ah, it takes a message.

Reply:
Should we add overloads to the classifier that work on Response (as opposed to on messages)?